packages/fx-core/resource/package.nls.json

```diff
@@ -187,6 +187,13 @@
   "plugins.spfx.questions.versionCheck.title": "Checking Node.js and NPM versions...",
   "plugins.spfx.questions.webpartName.error.duplicate": "The folder %s already exists. Choose a different name for your component.",
   "plugins.spfx.questions.webpartName.error.notMatch": "%s doesn't match pattern: %s",
+  "plugins.spfx.questions.packageSelect.title": "SharePoint Framework",
+  "plugins.spfx.questions.packageSelect.placeholder": "Select an option for scaffolding",
+  "plugins.spfx.questions.packageSelect.useGlobalPackage.withVersion.label": "Use globally installed SPFx (%s)",
+  "plugins.spfx.questions.packageSelect.useGlobalPackage.noVersion.label": "Use globally installed SPFx",
```

Comment:
Please check whether this "SPFx" is publicly communicated to end users/developers. If not through their official documents or sources, we should always avoid using acronyms, but to use the full name instead whenever possible.

Reply:
The finding will apply to all occurences of SPFx.

Reply:
Updated to "SharePoint Framework" as what spfx doc uses.

Reply:
synced offline and we will confirm with SPFx team about whether "SPFx" is enough.